tests/unit/symbol/test_symbol_graph.py

```diff
@@ -16,12 +18,15 @@ def sync_context(symbol_graph_static_test):  # noqa
     return synchronization_context
 
 
+@pytest.mark.skip("This test works only when in isolation from the other test")
```

Comment:
This test currently works, why would we need to start skipping? This is generally a sign something isn't acting the way we want it to.